lib/modules/touch.js

```diff
@@ -0,0 +1,50 @@
+import { contentLoaded } from '../utils';
+
+export default (shower) => {
+    contentLoaded(() => {
+        let action = false;
+        let clickable = false;
```

Comment:
Is is possible to merge these 2 into 1 boolean with more descriptive name?

Reply:
Now I rename `action` to `exitFullScreen`. Maybe it make give more information